packages/fast-usdc/src/exos/liquidity-pool.js

```diff
@@ -89,60 +121,149 @@ export const prepareLiquidityPoolKit = (zone, zcf, USDC, tools) => {
      */
     (shareMint, node) => {
       const { brand: PoolShares } = shareMint.getIssuerRecord();
-      const proposalShapes = makeProposalShapes({ USDC, PoolShares });
+      const proposalShapes = makeProposalShapes({
+        USDC,
+        PoolShares,
+      });
       const shareWorth = makeParity(makeDust(USDC), PoolShares);
       const { zcfSeat: poolSeat } = zcf.makeEmptySeatKit();
-      const shareWorthRecorderKit = tools.makeRecorderKit(node, RatioShape);
+      const { zcfSeat: feeSeat } = zcf.makeEmptySeatKit();
+      const poolMetricsRecorderKit = tools.makeRecorderKit(
+        node,
+        PoolMetricsShape,
+      );
+      /** used for `checkPoolBalance` invariant */
+      const encumberedBalance = makeEmpty(USDC);
```

Comment:
Perhaps expand to say: poolSeat USDC allocation plus outstanding borrows.

I wonder if this docstring would get associated with `state.encumberedBalance` if it were moved down to the `return` statement.

Reply:
Perhaps expand to say: ~~poolSeat USDC allocation plus outstanding borrows.~~ aka outstanding borrows